server/map_rpc_server.go

```diff
@@ -217,6 +232,11 @@ func (t *TrillianMapServer) SetLeaves(ctx context.Context, req *trillian.SetMapL
 			req.MapId,
 			writeRev,
 			hasher, func(ctx context.Context, f func(context.Context, storage.MapTreeTX) error) error {
+				if t.opts.UseSingleTransaction {
+					glog.V(1).Infof("EXP: using single tx")
```

Comment:
Maybe drop the `EXP: ` prefix and possibly reword for clarity e.g. `("using enclosing tx for sub-operation %d", txRolledUp)`

Reply:
done